### ISSUE TITLE
LSTMencdec - doubling the channel number at output

### ISSUE THREAD

**Issue by Garethlomax** [msc-acse/acse-9-independent-research-project-Garethlomax#22]
C:\Users\Gareth\Anaconda3\lib\site-packages\torch\nn\modules\loss.py:443: UserWarning: Using a target size (torch.Size([1, 10, 1, 64, 64])) that is different to the input size (torch.Size([1, 10, 2, 64, 64])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size. return F.mse_loss(input, target, reduction=self.reduction)

error message from above - why is there an increase in channle size / number??

**Comment by Garethlomax** [comment]
Error in LSTMencdec forward method. Method was returning encoded format of the method, not the unfolded to single layer. also h is of size 1, 3, 64, 64. this is unusual and should be checked out - thought this bug was fixed. see other issue for more
